images/build/Dockerfiles/vso.Dockerfile

```diff
@@ -30,5 +30,16 @@ RUN buildDir="/opt/tmp/build" \
     && condaDir="/opt/oryx/conda" \
     && mkdir -p "$condaDir" \
     && cd $imagesDir/build/python/conda \
-    && cp -rf * "$condaDir" \
+    && cp -rf * "$condaDir"
+
+# Install Ruby
+ENV RUBY_BIN="opt/ruby/lts/bin"
```

Comment:
Since we want to reduce the number of layers that are part of a docker image, these instructions need to be modified. Please take a look at my Java PR for idea on how to make changes in `vso.Dockerfile`

Reply:
Sure, I made changes to it